src/web/lib/ArticleRenderer.tsx

```diff
@@ -167,17 +169,29 @@ export const ArticleRenderer: React.FC<{
 							</>
 						</Figure>
 					);
-				case 'model.dotcomrendering.pageElements.DocumentBlockElement':
+				case 'model.dotcomrendering.pageElements.DocumentBlockElement': {
```

Comment:
Can we remove the enclosing braces here? Not because I especially care about this formatting but because all the other case statements in this switch don't use them and so this could present an issue later on if someone did a multi select during a refactor and wasn't expecting this pattern.